requirements.txt

```diff
@@ -10,6 +10,18 @@ Cartopy>=0.22.0
 pyproj>=3.4.1
 tueplots>=0.0.8
 plotly>=5.15.0
+torch-geometric>=2.5.2
+loguru>=0.7.2
+xarray>=2024.3.0
+zarr>=2.17.2
+dask>=2024.4.2
 
 # for dev
 pre-commit>=2.15.0
+codespell>=2.0.0
```

Comment:
these deps for linting are already defined and pinned in `.pre-commit-config.yaml`, it shouldn't be necessary to include them here too. Did you have problems with `pre-commit`?

Reply:
I did, but it must have been something else, because it works now without these dependencies.